extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -18,37 +21,24 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 		return true;
 	}
 
-	$html .= Xml::element( 'h2', array(), wfMsg( 'imagereview-imagepage-header' ) );
-
-	$reviews = array();
-	$headers = array(
-		wfMessage('imagereview-imagepage-table-header-reviewer')->text(),
-		wfMessage('imagereview-imagepage-table-header-state')->text(),
-		wfMessage('imagereview-imagepage-table-header-time')->text(),
-	);
-	$where = array(
-			'wiki_id' => $wgCityId,
-			'page_id' => $imagePage->getId(),
-	);
-
-	$dbr = wfGetDB( DB_SLAVE, array(), $wgExternalDatawareDB );
+	$html .= Xml::element( 'h2', [], wfMessage( 'imagereview-imagepage-header' )->escaped() );
 
-	$res = $dbr->select(
-		'image_review_stats',
-		'*',
-		$where
-	);
+	$headers = [
+		wfMessage( 'imagereview-imagepage-table-header-reviewer' )->text(),
+		wfMessage( 'imagereview-imagepage-table-header-state' )->text(),
+		wfMessage( 'imagereview-imagepage-table-header-time' )->text(),
+	];
 
-	if ( $dbr->numRows( $res ) == 0 ) {
-		//check if image is in the queue at all!
+	$dbr = wfGetDB( DB_SLAVE, [], $wgExternalDatawareDB );
 
-		$imgCurState = $dbr->selectField(
-			'image_review',
-			'state',
-			$where	
-		);
+	//TODO: community switch
+	$reviews =
+		fetchReviewHistoryFromService( $wgCityId, $imagePage->getID(), $imagePage->getTitle()->getLatestRevID() );
+	//$reviews = fetchReviewHistory( $dbr, $wgCityId, $imagePage->getID() );
+	if ( empty( $reviews ) ) {
 
-		if ( false === $imgCurState ) {
+		// TODO: consider if in the new flow this also should be checked
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)